plugin-core/src/main/kotlin/io/github/simonschiller/permissioncheck/config/TaskConfigurator.kt

```diff
@@ -43,7 +43,7 @@ abstract class TaskConfigurator {
         extension: PermissionCheckExtension,
         variantName: String
     ): TaskProvider<PermissionCheckTask> {
-        val taskName = "check${variantName.capitalize(Locale.ROOT)}Permissions"
+        val taskName = "check${variantName.replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.ROOT) else it.toString() }}Permissions"
```

Comment:
This was a warning from android studio and the fix applied also by the same android studi